### PR TITLE
Bug wikipage author

### PR DESCRIPTION
**Pull request by curioussavage**
* [ ] fixes bug around handling missing author property on revisions

Also broke the `makeContent` method into several smaller ones.

### CODE REVIEW

**Comment by curioussavage** [comment]
:eyeglasses: @nramadas  or @schwers

**Comment by curioussavage** [comment]
:eyeglasses: @ajacksified or @schwers or @nramadas

**Comment by curioussavage** [comment]
related to (but not dependent on)https://github.com/reddit/snoode/pull/74

**Comment by curioussavage** [comment]
so for the double eyeglasses, github wasn't showing the first ones :frowning:

**Comment by ajacksified** [comment]
:fish:

File: test/views/pages/wikipage.jsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import chai from 'chai';
+import sinonChai from 'sinon-chai';
+
+import findText from '../../helpers/findtext';
+import renderWithProps from '../../helpers/renderWithProps';
+
+import { WikiPage, Loading } from '../../helpers/components';
+import { MESSAGES } from '../../../src/views/pages/wikipage';
+
+var expect = chai.expect;
```
Comment (schwers) [inline test/views/pages/wikipage.jsx]: `const`

Comment (ajacksified) [inline test/views/pages/wikipage.jsx]: on that note, all the `lets` below here should be consts too

**Comment by schwers** [comment]
:fish: after rebase

**Pull request merged**